### PR TITLE
Updated 2.0 Docs on Fixing Incomplete Rules

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- Updated 2.0 docs on Incomplete Rules (Fixes unresolved docs issues from PR #6845)

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/policies.mdx [hunk 1]
```diff
@@ -387,8 +387,8 @@ slots and active loops set by actions are defined consistently for all rules.
      - action: utter_venues_not_found
    ```
 
-   - if you want to deal with setting slot in a different rule or story,
-   you should add `wait_for_user_input: false` in the the end of the rule snippet.
+   If you want to deal with slot setting in a different rule or story,
+   you should add `wait_for_user_input: false` to the end of the rule snippet.
```
Comment (erohmensing) [inline docs/docs/policies.mdx:391]:
```suggestion
   If you want the slot setting to be handled by a different rule or story,
   you should add `wait_for_user_input: false` to the end of the rule snippet:
```

File: docs/docs/policies.mdx [hunk 2]
```diff
@@ -368,12 +368,12 @@ slots and active loops set by actions are defined consistently for all rules.
...
+   There are several possible ways to fix this rule.
 
-   - if you want to cover the case when `action_search_venues` could not find
-   anything and therefore `venues` should not be set,
+   In the case when `action_search_venues` can't find
+   a venue and the `venues` slot should not be set,
    you should explicitly set the value of the slot to `null`.
-   Then in the following story `RulePolicy` will predict `utter_venues_not_found`
+   The following story `RulePolicy` will predict `utter_venues_not_found`
    only if the slot `venues` is not set.
```
Comment (erohmensing) [inline docs/docs/policies.mdx:377]:
```suggestion
   only if the slot `venues` is not set:
```

File: docs/docs/policies.mdx [hunk 3]
```diff
@@ -348,7 +348,7 @@ policies:
    further information.
 * `check_for_contradictions` (default: `true`):
    Before training, the RulePolicy will perform a check to make sure that
-slots and active loops set by actions are defined consistently for all rules. 
+   slots and active loops set by actions are defined consistently for all rules.
    The following snippet is an example of an incomplete rule:
```
Comment (erohmensing) [inline docs/docs/policies.mdx:352]:
```suggestion
   The following snippet contains an example of an incomplete rule:
```

**Review by erohmensing — APPROVED** [review]

**Comment by tmbo** [comment]
should this target 2.x?

**Comment by erohmensing** [comment]
It should be closed, the PR mentioned above ported it to 2.0.x

**Pull request closed without merging**